codemods/utils/convert-css-object.ts

```diff
@@ -219,10 +218,38 @@ export const addProperty = (
   identifier,
   value,
   isSupported,
+  substitutionMap,
   comment?: string
 ) => {
-  const prefix = !isSupported ? `// ${TODO_RN_COMMENT}\n// ` : '';
-  const prop = j.property('init', j.identifier(prefix + identifier), j.literal(value));
+  let unsupportedComment = '';
+  // replace substitution back into comment.
+  function literalBasedOnSupported() {
+    if (!isSupported) {
+      const { foundExpression, pre, post } = getExpressionFromMap(substitutionMap, value);
+      if (foundExpression) {
+        const parsed = parseExpression(j, foundExpression, []);
+
+        if (pre || post) {
+          if (['rem', 'vw', 'vh', 'em'].includes(post)) {
+            unsupportedComment = `\n// ${post} are not supported in the interpolated value`;
+          }
+          return j.templateLiteral(
+            [
+              j.templateElement({ cooked: pre, raw: pre }, false),
+              j.templateElement({ cooked: post, raw: post }, false),
+            ],
+            [parsed.value]
+          );
+        }
+      }
+    }
+
+    return j.literal(value);
+  }
```

Comment:
oh this also makes it so the commented out code no longer has the `__4substition__` in it!

Reply:
Awesome!